testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractGroupMapperTest.java

```diff
@@ -86,7 +87,7 @@ protected UserRepresentation loginAsUserTwiceWithMapper(
         updateUser();
 
         logInAsUserInIDP();
-        assertLoggedInAccountManagement();
+        Assert.assertFalse(loginPage.isCurrent());
```

Comment:
Shouldn't this be updated to login to test app, and check that is opened rather the checking login page isn't current page